lib/puppet/parser/functions/default_content.rb

```diff
@@ -1,28 +1,28 @@
-module Puppet::Parser::Functions
-  Puppet::Parser::Functions.newfunction(:default_content,
-                                        :type => :rvalue,
-                                        :doc => <<-'ENDOFDOC'
-  Takes an optional content and an optional template name and returns the
-  contents of a file.
+Puppet::Parser::Functions.newfunction(:default_content,
+                                      :type => :rvalue,
+                                      :doc => <<-'ENDOFDOC'
+Takes an optional content and an optional template name and returns the
+contents of a file.
 
-  Examples:
+Examples:
 
-      $config_file_content = default_content($config_file_string, $config_file_template)
-      file { '/tmp/x':
-        ensure  => 'file',
-        content => $config_file_content,
-      }
-  ENDOFDOC
-  ) do |args|
-    content = args[0]
-    template_name = args[1]
+    $config_file_content = default_content($config_file_string, $config_file_template)
```

Comment:
Perhaps the example should set `$config_file_string` and `$config_file_template`, so users understand what format is expected in each argument. Alternatively, use the example variable names `$file_content` and `$template_location` as I believe that maps better to the expectation.